parser_genomes.py

```diff
@@ -69,11 +70,41 @@ def setup_directories(family, sample_list, filepath, step):
     if len(ped) > 1:
         print(f"Multiple ped files found: {ped}. Exiting!")
         exit()
+    if len(ped) == 0:
+        print(f"No ped files found: {ped}. Exiting!")
+        break
     if len(ped) == 1 and os.path.isfile(config):
         ped = ped[0]
-        replace = 's+ped: ""+ped: "{}"+'.format(ped)
-        cmd = ["sed", "-i", replace, config]
-        subprocess.check_call(cmd)
+        pedi = pd.read_csv(
+            ped,
+            sep=" ",
+            header=None,
+            names=["fam_id", "individual_id", "pat_id", "mat_id", "sex", "phenotype"]
+            )
+        for row in range(len(pedi)): #check whether any of the information is numeric (ie. unwanted information)
```

Comment:
For future reference, there are [faster](https://realpython.com/fast-flexible-pandas/) ways to iterate through pandas data frames, that also have simpler syntax. Speed isn't much of an issue here because the pedigree file is so small, at least.